src/window/hotkey_duplicate.c

```diff
@@ -0,0 +1,90 @@
+#include "hotkey_duplicate.h"
```

Comment:
I think it's better to expand the `plain_message_dialog` with an extra (optional) text parameter rather than to duplicate the code for this dialog.

Reply:
Does pure C allow optional parameters in function declaration?

Reply:
Yes, a second function is best in this case.
Suggestion: name the function `window_plain_message_dialog_show_with_extra` the parameter `extra`?

Reply:
Done.